readthedocs/projects/forms.py

```diff
@@ -706,10 +706,15 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # Disable domain manipulation on Update, but allow on Create
-        instance = getattr(self, 'instance', None)
-        if instance and instance.pk:
+        if self.instance.pk:
```

Comment:
I hope this works :) I don't remember if there can be instance=None on a creation form? But I guess you've checked that one :)

Reply:
This attribute was always defined in my local testing, I think it's None if explicitly set to None :man_shrugging: